kratos/tests/cpp_tests/strategies/strategies/test_explicit_strategies.cpp

```diff
@@ -75,6 +75,18 @@ namespace Testing
             r_node.FastGetSolutionStepValue(REACTION_FLUX) = 37.5 - 3.5 * aux;
         }
 
+        void EquationIdVector(
+            EquationIdVectorType& rEquationIdVector,
+            const ProcessInfo& rCurrentProcessInfo) const override
+        {
+            unsigned int local_size = GetGeometry().PointsNumber();
+            rEquationIdVector.resize(local_size);
+            for (unsigned int i=0; i<local_size; i++)
+            {
+                rEquationIdVector[i] = GetGeometry()[i].GetDof(TEMPERATURE).EquationId();
+            }
```

Comment:
```suggestion
            if (rEquationIdVector.size() != 1) {
                rEquationIdVector.resize(1)
            }
            rEquationIdVector[0] = GetGeometry()[0].GetDof(TEMPERATURE).EquationId();
```
This is a single node with one DOF element so there is no need for the loop.